packages/backend/src/dbt/dbtCliClient.ts

```diff
@@ -207,14 +252,27 @@ export class DbtCliClient implements DbtClient {
         }
     }
 
-    private async loadDbtTargetArtifact(filename: string): Promise<any> {
-        const targetDir = await this._getTargetDirectory();
-        const fullPath = path.join(
-            this.dbtProjectDirectory,
-            targetDir,
-            filename,
-        );
-        return DbtCliClient.loadDbtFile(fullPath);
+    private async loadDbtTargetArtifact(
+        version: DbtManifestVersion,
+        filename: string,
+    ): Promise<any> {
+        switch (version) {
+            case DbtManifestVersion.V9:
+                return DbtCliClient.loadDbtFile('./target/manifest.json');
```

Comment:
can't the target dir be configured in dbt 1.5 ? Or do we need to change the default value in `_getTargetDirectory` depending on the version ?

Reply:
yes, you can change the target on 1.5 too, I'll move the logic to the _getTargetDirectory method then

Reply:
Done